src/main/app/e2e/support/worlds/world.ts

```diff
@@ -13,7 +14,7 @@ export class CustomWorld extends World {
 
     async init() {
         this.browser = await playwright.chromium.launch({
-            headless: false,
+            headless: true,
```

Comment:
might be nice to load this from an environment property, so that we have the option to show the way this works (in a demo)